### PR TITLE
Add script to help generate PR messages

### PR DESCRIPTION
**Pull request by dnfield**
This will massage the output of `git log` for a commit message to flutter/flutter.  Simply meant to help with manual rolls - normally the autoroller takes care of this.

### CODE REVIEW

File: tools/engine_roll_pr_desc.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [[ $1 == '' ]]; then
+  echo 'Usage: engine_roll_pr_desc.sh <from git hash>..<to git hash>'
+  exit 1
+fi
+git log --oneline --no-merges --no-color $1 | sed 's/^/flutter\/engine@/g' |  sed -e 's/(\(#[0-9]*)\)/\(flutter\/engine\1/g'
```
Comment (cbracken) [inline tools/engine_roll_pr_desc.sh]: Add an EOF newline

**Comment by cbracken** [comment]
Oh nice -- I have a vim macro for this that I can deprecate :)

File: tools/engine_roll_pr_desc.sh (hunk 1 above)
Comment (cbracken) [reply tools/engine_roll_pr_desc.sh]: [Rationale](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

Comment (dnfield) [reply tools/engine_roll_pr_desc.sh]: Done

**Pull request merged**